src/Tizen.NUI/src/public/Visuals/VisualObject/ImageVisual.cs

```diff
@@ -174,6 +174,28 @@ public bool OrientationCorrection
             }
         }
 
+        /// <summary>
+        /// Gets or sets whether to automatically reload the image as the visual size<br />
+        /// </summary>
+        /// <remarks>
+        /// If we set this value as true, <see cref="DesiredWidth"/> and <see cref="DesiredHeight"/> will be invalidated.
+        /// </remarks>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public bool SynchronousSizing
+        {
+            set
+            {
+                UpdateVisualProperty((int)Tizen.NUI.ImageVisualProperty.SynchronousSizing, new PropertyValue(value), true);
+            }
+            get
+            {
+                bool ret = true;
```

Comment:
Is this ok if the propertyValue is null, then true is returned.

Reply:
Opps.. It should be false. Thank you for your notify it. I'll fix it.